### DISCUSSION TITLE
Run Next.js app with forever

### DISCUSSION THREAD

**Discussion by ashishpixels** [vercel/next.js#14616] [Help]
How do I start the project that I've built using `forever`.

- `yarn next-start` works effortlessly to start it on server.
- But I want to keep the project running forever.

Using "forever start /node_modules/.bin/next" gives an error: `script /node_modules/.bin/next does not exist`
Because, obviously, it couldn't find a script to run.

So, how do I do it ?

**Comment by IamLizu** [comment]
Can you show any sample? maybe you code repo. 
Meanwhile, try deleting your `node_modules` and installing packages again with `yarn`.

**Reply by ashishpixels** [reply]
@IamLizu  Thanks for the reply. Much appreciated. I did try doing this, which didn't solve the problem. Then eventually found the solution some time later.
The issue was the way my project was built, which was made for both Gatsby and Next at the same time. so it had a different structure than the usual Next code. The "pages" folder was in a different directory which is needed to be pointed out while starting a Next code on server. I just had to point to the right "pages" directory while using forever. Since this was taken care of for the "yarn next-start" command in my code, I assumed forever wouldn't cause any issues.
anyway, thanks again for the reply.
